docs/src/user-guide/configuring/plugins.md

```diff
@@ -1,93 +1,25 @@
 ---
-title: Plugins
+title: Plugins & Parsers
+layout: doc
 eleventyNavigation:
     key: configuring plugins
     parent: configuring
-    title: Configuring Plugins
+    title: Configuring Plugins & Parsers
     order: 4
 
 ---
 
-## Specifying Parser
+You can extend ESLint with plugins in a variety of different ways. Plugins can include:
 
-By default, ESLint uses [Espree](https://github.com/eslint/espree) as its parser. You can optionally specify that a different parser should be used in your configuration file so long as the parser meets the following requirements:
+* Custom processors to extract JavaScript code from other kinds of files or preprocess code before linting.
+* Custom rules to validate if your code meets a certain expectation, and what to do if it does not meet that expectation.
+* Other custom configurations using ESLint's built-in features.
```

Comment:
```suggestion
* Custom configurations.
```

There are no configurations before this item, so "Other" sounds confusing. Also, custom configurations can use both built-in features and plugins (e.g., a configuration can enable some rules from the plugin and some built-in rules).